target/linux/ath79/image/generic.mk

```diff
@@ -979,6 +979,21 @@ define Device/sitecom_wlr-7100
 endef
 TARGET_DEVICES += sitecom_wlr-7100
 
+define Device/sitecom_wlr-8100
+  ATH_SOC := qca9558
+  DEVICE_VENDOR := Sitecom
+  DEVICE_MODEL := WLR-8100
+  DEVICE_VARIANT := v1 002
```

Comment:
I'd also remove the variant here, but you could add an ALT0 entry for the X8 ACxxxx name in DTS model here.

Reply:
It should be fixed now